tests/components/insteon/test_config_flow.py

```diff
@@ -173,7 +116,7 @@ async def test_fail_on_existing(hass: HomeAssistantType):
         context={"source": config_entries.SOURCE_USER},
     )
     assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
-    assert result["reason"] == "already_configured"
+    assert result["reason"] == "single_instance_allowed"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)